students/gyuchan/users/views.py

```diff
@@ -1,3 +1,55 @@
-from django.shortcuts import render
+import json
+import re
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from users.models import User
+
+class UsersView(View):
```

Comment:
```suggestion
class SignUpView(View):
```

UsersView라고 하면 그 기능에 대해 명확하게 설명할 수 없을 것 같습니다.
마치 users 정보를 등록하거나 불러오는 것 처럼 보이기도 하네요.
기능에 맞게 수정 부탁드려요~!